extract/extract/extractors/zalando.py

```diff
@@ -180,19 +164,19 @@ def _get_sustainability(
             beautiful_soup=beautiful_soup,
             title_attr="certificate__title",
             description_attr="certificate__description",
-            headline=headline,
```

Comment:
Because this string is used multiple times, it is better to use a parameter with default value.